src/components/Footer/styles.scss

```diff
@@ -5,14 +5,16 @@
   height: 100%;
   background-color: colors.$black;
   color: colors.$white;
-  z-index: 1;
+  z-index: -1;
   position: relative;
 }
+
```

Comment:
Nitpick: Let's try to avoid adding new lines to commits that don't need to be there to keep our code readable to reviews and make it clear where the real changes to our code are.

You can check the diff with the highlighted lines in a pull request (green for added, red for deleted, orange for modifed), on Github Desktop, or in the VSCode sidebar next to the line of code if it hasn't been staged yet.

Reply:
Sorry to be picky on this but we didn't actually delete all of the newlines in the file 😞  If you delete the empty lines and commit the changes, the green lines here should disappear from this file on the pull request.